components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/impl/SingleApiServiceImpl.java

```diff
@@ -31,7 +17,7 @@ public Response runSingleSimulation(String singleEventConfiguration) {
             return Response.ok().header("Access-Control-Allow-Origin", "*")
                     .entity(new ResponseMapper(Response.Status.OK, "Single Event simulation started successfully"))
                     .build();
-        } catch (InvalidConfigException | InsufficientAttributesException | ResourceNotFoundException e) {
+        } catch (ResourceNotFoundException e) {
```

Comment:
I think resources not found is not the best supper type to use. How about having something like event simulation exception and defining others as sub type of that?

Reply:
@suhothayan , I will create another exception class called SimulationException and catch all three exceptions (InvalidConfigException, InsufficientAttributesException, ResourceNotFoundException) and populate the active and inactive maps accordingly.